setup.py

```diff
@@ -27,10 +27,11 @@
     keywords='Telegram messaging database',
     packages=find_packages(),
     install_requires=[
-        'tqdm', 'telethon-aio', 'appdirs',
+        'tqdm', 'telethon-aio', 'appdirs', 'PySocks',
```

Comment:
Not sure if this should be a hard dependency. We could do something like `pip install telegram-export[proxy]`, so that it doesn't need to be installed unless you need it.